honeycombio/resource_trigger_test.go

```diff
@@ -208,6 +218,48 @@ resource "honeycombio_trigger" "test" {
 }`, dataset, dataset, frequency)
 }
 
+func testAccTriggerConfigWithAlertType(dataset string, alertType string) string {
```

Comment:
I don't think this gets called? Cool to just rely on what you've done above and remove it